bitcoin-core-dev-tech/2019-06-05-wallet-architecture.es.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Arquitecturas de las wallets (2019-06-05)
+transcript_by: Bryan Bishop
+translation_by: Blue Moon
+categories: ['core-dev-tech']
+tags: ['wallet', 'bitcoin core']
+---
```

Comment:
Thank you! There are some changes in the format of the transcript's metadata, as explained in the updated [README](https://github.com/bitcointranscripts/bitcointranscripts#contribution).

You can also see how the original file's metadata have changed
https://github.com/bitcointranscripts/bitcointranscripts/blob/17d07a13e1d238342b862618ddd56f43e09f98cb/bitcoin-core-dev-tech/2019-06-05-wallet-architecture.md?plain=1#L1-L8

Reply:
Thank you very much, updated.